js/src/components/shipping-rate-section/minimum-order-card/edit-minimum-order-form-modal.js

```diff
@@ -4,6 +4,7 @@
 import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { Form } from '@woocommerce/components';
+import { noop } from 'lodash';
```

Comment:
:nail_care:/:question:
I always have this recurring doubt, do we need to import a full blown library (with all dependency costs) for something as simple as `const noop = () => {};`

Reply:
I'm feeling like we don't need to worry about it that much because we are loading one shared copy of lodash via DEWP anyway. And I also think using `import { noop } from 'lodash'` is better than having to define `const noop = () => {};` in many files.

Reply:
Given the problems, we had with DEWP ;) it would be nice to have fewer reasons to use it ;)